packages/data/walkthroughs/9.9.9.json

```diff
@@ -1555,6 +1555,326 @@
           "nextNavigateTo": "P11"
         }
       ]
+    },
+    "P11": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "How many <defined-term>dwelling units</defined-term> are in this <defined-term>building</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "1-2",
+          "answerValue": "2"
+        },
+        {
+          "answerDisplayText": "Greater than 2",
+          "answerValue": "3"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P12"
+        }
+      ]
+    },
+    "P12": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Is this <defined-term>dwelling unit</defined-term> provided with at least two separate <defined-term>means of egress</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P12",
+          "answerValue": "true",
+          "nextNavigateTo": "R10"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P13"
+        }
+      ]
+    },
+    "P13": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Does this <defined-term>dwelling unit</defined-term>’s egress door open onto a <defined-term>public corridor</defined-term> or an exterior passageway?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P13",
+          "answerValue": "false",
+          "nextNavigateTo": "P17"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P14"
+        }
+      ]
+    },
+    "P14": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "Where the egress door opens onto the corridor or exterior passageway, is it possible to go in opposite directions to 2 separate <defined-term>exits</defined-term>?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P14",
+          "answerValue": "true",
+          "nextNavigateTo": "P17"
+        },
+        {
+          "nextLogicType": "greaterThan",
```

Comment:
@laurenrevere Yeah to me this could go any which way but I like this solution since, as you said, it matches the SME document better. We really just need a value to key off of in a specific way.